js/plugins/compat-oai/README.md

```diff
@@ -142,7 +142,6 @@ import { ModelInfo } from 'genkit/model';
 import openAI from '@genkit-ai/compat-oai';
 
 const modelInfo: ModelInfo = {
```

Comment:
this is optional, right? if not specified a generic (everything supported) model info would be used, right?

Reply:
it is optional but I don't set a default explicitly in this plugin. I thought the framework had a default by itself.... I might be wrong.